src/software/world/team_test.cpp

```diff
@@ -551,3 +551,47 @@ TEST_F(TeamTest, update_timestamp_history_from_team_update)
     EXPECT_EQ(three_seconds_future, team.getTimestampHistory()[0]);
     EXPECT_EQ(4, team.getTimestampHistory().size());
 }
+
+TEST_F(TeamTest, get_all_robots_except_goalie)
+{
+    Team team = Team(Duration::fromMilliseconds(2000));
+    Team teamWithoutGoalie = Team(Duration::fromMilliseconds(2000));
```

Comment:
For variables, they should follow `snake_case` whereas functions should be `camelCase`.

Reply:
please review the [code style guide](https://github.com/UBC-Thunderbots/Software/blob/master/docs/code-style-guide.md)